Traceback (most recent call last):
  File "/workspace/ACCV_2024/PRCSL/src/main.py", line 312, in <module>
    main()
  File "/workspace/ACCV_2024/PRCSL/src/main.py", line 260, in main
    appr.train(t, trn_loader[t], val_loader[t])
  File "/workspace/ACCV_2024/PRCSL/src/approach/incremental_learning.py", line 55, in train
    self.train_loop(t, trn_loader, val_loader)
  File "/workspace/ACCV_2024/PRCSL/src/approach/der.py", line 62, in train_loop
    super().train_loop(t, trn_loader, val_loader)
  File "/workspace/ACCV_2024/PRCSL/src/approach/incremental_learning_der.py", line 46, in train_loop
    self.train_epoch(t, trn_loader)
  File "/workspace/ACCV_2024/PRCSL/src/approach/der.py", line 69, in train_epoch
    for i, (images, targets, no_aug_images) in enumerate(trn_loader):
  File "/opt/conda/envs/prcsl/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 681, in __next__
    data = self._next_data()
  File "/opt/conda/envs/prcsl/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 721, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/opt/conda/envs/prcsl/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 49, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/opt/conda/envs/prcsl/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 49, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/workspace/ACCV_2024/PRCSL/src/datasets/base_dataset.py", line 24, in __getitem__
    x = Image.open(self.images[index]).convert('RGB')
  File "/opt/conda/envs/prcsl/lib/python3.8/site-packages/PIL/Image.py", line 3431, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/workspace/ACCV_2024/PRCSL/data/ham10000/train/bkl/trans__196_8470343.jpg'
